src/refactor_nrepl/middleware.clj

```diff
@@ -54,7 +54,7 @@
   `(with-errors-being-passed-on ~transport ~msg
      (config/with-config ~msg
        (transport/send ~transport
-                       (response-for ~msg ~(apply hash-map :status :done kvs))))))
```

Comment:
looks like all reply-functions below call this macro with `:status :done` (all except `find-used-locals-reply`). if so, maybe it would be better to get rid of status from here and use `reply` in `find-symbol-reply` which would simplify the code and make it more DRY-ish?